### DISCUSSION TITLE
UPDATE Button doesnt work for me.

### DISCUSSION THREAD

**Discussion by githubUser01946** [vladmandic/sdnext#3014] [General]
![image](https://github.com/vladmandic/automatic/assets/117453319/d7abd1f7-1353-463e-960b-bc1a3cd8cc79)

I click update and it will say : `Extension updated | the extension I am updating | Restart required`, I click the Apply changes button but the update button remains in place, it isnt changing to uninstall or anything like the other buttons. 

It is doing this with all the awaiting updates.

**Comment by vladmandic** [comment]
> Restart required

**Reply by githubUser01946** [reply]
When pressing apply changes it does that or appears to but I pressed restart and it still says it.

I've tried 3 different browsers.
